less/home-page.less

```diff
@@ -45,10 +53,18 @@
     padding: 50px 60px;
     display: flex;
     justify-content: center;
-
+    @media @phone {
+      flex-direction: column-reverse;
```

Comment:
nice work with changing up the positioning using column-reverse in the media query